.github/workflows/optimize-release-optimize-c8-only.yml

```diff
@@ -122,6 +131,7 @@ jobs:
             echo "IS_ALPHA=${{ steps.release-type.outputs.is_alpha }}"
             echo "IS_RC=${{ steps.release-type.outputs.is_rc }}"
             echo "IS_DRY_RUN=${{ github.event.inputs.IS_DRY_RUN || 'true' }}"
+            echo "SHOULD_PUSH=${{ steps.release-type.outputs.should_push }}"
```

Comment:
why do we need a new property rather than just reusing (and inverting) the `IS_DRY_RUN` property?